mkosi/__init__.py

```diff
@@ -2241,6 +2242,95 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        from _emerge.actions import action_build, action_sync, load_emerge_config
```

Comment:
Is this a python API to emerge? Current practice in mkosi is to not depend on any python library dependencies. Can you call emerge via the command line instead?

Reply:
I think we could maybe loosen this as long as this is in distro-specific code, when its confined to a function, and significantly shortens our work. When people build emerge it should be little overhead to also build the python bindings.

Reply:
shell version:
https://github.com/systemd/mkosi/pull/691

Reply:
as commented below I'd be fine with using python APIs of packaging scripts, see below for my reasoning.

Reply:
missed this one. great, i will continue with python API then.